src/platforms/android/manual-configuration.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: Manual Configuration
+sidebar_order: 2
+description: "Learn how to manually configure the Sentry Android SDK."
+---
+<PlatformContent includePath="getting-started-primer" />
+
+<PlatformSection noGuides notSupported={["android", "dart", "elixir", "flutter", "perl", "react-native", "unity", "unreal"]}>
+
+<Alert level="info" title="Using a framework?">
+
+Get started using a guide listed in the right sidebar.
+
+</Alert>
+
+</PlatformSection>
+
+## Installation
+
+If you don't want the Sentry Gradle plugin to install the Sentry SDK automatically for you, you can define the dependency directly to your `build.gradle` file of your app module.
+
+```groovy {filename:app/build.gradle}
+// Make sure mavenCentral is there.
+repositories {
+    mavenCentral()
+}
+
+// Enable Java 1.8 source compatibility if you haven't yet.
+android {
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+}
+
+// Add Sentry's SDK as a dependency.
+dependencies {
+    implementation 'io.sentry:sentry-android:{{ packages.version('sentry.java.android', '4.2.0') }}'
+}
+```
+
+```kotlin {filename:app/build.gradle.kts}
+// Make sure mavenCentral is there.
+repositories {
+    mavenCentral()
+}
+
+// Enable Java 1.8 source compatibility if you haven't yet.
+android {
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+}
+
+// Add Sentry's SDK as a dependency.
+dependencies {
+    implementation("io.sentry:sentry-android:{{ packages.version('sentry.java.android', '4.2.0') }}")
+}
+```
+
+<Note>
+
+[NDK integration](/platforms/android/using-ndk/) is packed with the SDK and requires API level 16, though other levels are supported.
+
+If you are using multiple Sentry dependencies, you can add a [bill of materials](/platforms/android/configuration/bill-of-materials) to avoid specifying the version of each dependency.
+
+</Note>
+
+## Configuration
+
+<Note>
+
+Don't already have an account and Sentry project established? Head over to [sentry.io](https://sentry.io/signup/), then return to this page.
+
+</Note>
+
+<PlatformContent includePath="getting-started-config" />
+
```

Comment:
```suggestion
## Configuration

<PlatformContent includePath="getting-started-config" />

<Note>

Don't already have an account and Sentry project established? Head over to [sentry.io](https://sentry.io/signup/), then return to this page.

</Note>

```
I think it makes more sense for this note to come after we give them configuration instructions. It's a bit distracting if it's the first thing you see.